pennylane/tape/tape.py

```diff
@@ -239,14 +239,26 @@ def expand_tape_state_prep(tape, skip_first=True):
 
     Args:
         tape (QuantumScript): The tape to expand.
-        skip_first (Bool): If ``True``, will not expand a StatePrepBase operation if
+        skip_first (Bool): If ``True``, will not expand a ``StatePrepBase`` operation if
```

Comment:
```suggestion
        skip_first (bool): If ``True``, will not expand a ``StatePrepBase`` operation if
```
Slightly more consistent with the rest of the docs (mostly)